components/user_settings/display/index.ts

```diff
@@ -36,6 +37,9 @@ function mapStateToProps(state: GlobalState) {
     const lockTeammateNameDisplay = getLicense(state).LockTeammateNameDisplay === 'true' && config.LockTeammateNameDisplay === 'true';
     const configTeammateNameDisplay = config.TeammateNameDisplay as string;
 
+    // calling it once here avoids multiple calls to the function in the return
```

Comment:
Nit: This is clear. Why the comment?

Reply:
It's just a habit of mine to document stuff inline